modules/nf-core/untar/main.nf

```diff
@@ -11,50 +11,49 @@ process UNTAR {
     tuple val(meta), path(archive)
 
     output:
-    tuple val(meta), path("$untar"), emit: untar
-    path "versions.yml"            , emit: versions
+    tuple val(meta), path("$prefix"), emit: untar
+    path "versions.yml"             , emit: versions
 
     when:
     task.ext.when == null || task.ext.when
 
     script:
     def args  = task.ext.args ?: ''
     def args2 = task.ext.args2 ?: ''
-    untar     = archive.toString() - '.tar.gz'
+    prefix = task.ext.prefix ?: "${meta.id}"
```

Comment:
And nf-core said: Equal signs shall be aligned
```suggestion
    prefix    = task.ext.prefix ?: "${meta.id}"
```

Reply:
Ahh, that makes sense. I'm still learning, thank you. I'll fix that then.

Reply:
There was just an error with a process select in the kraken subworkflow, which was causing a prefix declaration to apply to the tar process. This hadn't mattered previously (since tar was not using prefix), but does now. Fixed by increasing the selectivity.